src/python_testing/TC_RVCCLEANM_2_2.py

```diff
@@ -94,13 +94,23 @@ async def test_TC_RVCCLEANM_2_2(self):
         self.is_ci = self.check_pics("PICS_SDK_CI_ONLY")
         if self.is_ci:
             app_pid = self.matter_test_config.app_pid
-            if app_pid == 0:
-                asserts.fail("The --app-pid flag must be set when PICS_SDK_CI_ONLY is set.c")
-            self.app_pipe = self.app_pipe + str(app_pid)
+            if app_pid != 0:
+                self.is_ci = True
+                self.app_pipe = self.app_pipe + str(app_pid)
 
-        asserts.assert_true(self.check_pics("RVCCLEANM.S"), "RVCCLEANM.S must be supported")
-        asserts.assert_true(self.check_pics("RVCRUNM.S.A0000"), "RVCRUNM.S.A0000 must be supported")
-        asserts.assert_true(self.check_pics("RVCRUNM.S.A0001"), "RVCRUNM.S.A0001 must be supported")
+        # replaces the RVCRUNM attributes from PICS file
+        RVCRun_cluster = Clusters.RvcRunMode
+        attributes = Clusters.RvcRunMode.Attributes
+        RVCRun_attr_list = attributes.AttributeList
+        attribute_list = await self.read_single_attribute_check_success(endpoint=self.endpoint, cluster=RVCRun_cluster, attribute=RVCRun_attr_list)
+        supported_modes_attr_id = attributes.SupportedModes.attribute_id
+        current_mode_attr_id = attributes.CurrentMode.attribute_id
+
+        if supported_modes_attr_id not in attribute_list:
+            asserts.fail("Supported modes needs to be supported attribute")
+
+        if current_mode_attr_id not in attribute_list:
+            asserts.fail("Current mode needs to be supported attribute")
```

Comment:
I guess these can be removed as well now. Same in the following scripts.

Reply:
Thank you, I have removed these conditional checks for current mode and supported mode attributes from the RVC Clean and Run test modules.